src/components/InventoryTabs/ImmutableDevices/EdgeDevicesTab.js

```diff
@@ -0,0 +1,33 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import AsyncComponent from '@redhat-cloud-services/frontend-components/AsyncComponent';
+import { resolveRelPath } from '../../../Utilities/path';
+import {
+  getNotificationProp,
+  manageEdgeInventoryUrlName,
+} from '../../../Utilities/edge';
+import { useLocation, useNavigate } from 'react-router-dom';
+import { useDispatch } from 'react-redux';
+
+const ImmutableDevicesTab = () => {
```

Comment:
nit-pick: wouldn't it be bad to align the module and filename naming? So that we use `EdgeDevicesTab` or `ImmutableDevicesTab` everywhere?

Reply:
Actually, most of the apps are going to have consolidated tabs in the future. Thus, this is better approach